src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ContextAwareness API
+layout: default
+category: API
+---
+
+The ContextAwareness API of Infusion provides a powerful suite of grades and utilities that allow a component to be responsive to aspects of its contexts in a flexible and open-ended way.
+For lightweight cases of context adaptation, the [`distributeOptions`](IoCSS.md) scheme suffices, in that it enables a component to be responsive to broadcasts "advising" it from around the
```

Comment:
This opening paragraph is too long and too all-encompassing. Our stylesheet renders the first paragraph in larger font, making it a quick at-a-glance into to what this page is about. I suggest ending the opening paragraph for this page after that first sentence.
The rest of this paragraph uses words and phrases that are not explained, and so are confusing to the reader (e.g. "adaptation", "two or more "axes" or "dimensions" of adaptation"). As Colin suggests below, an example of what the heck this is all about might be helpful before going into details.